apps/web/src/components/(dashboard)/common/command-menu.tsx

```diff
@@ -0,0 +1,112 @@
+'use client';
+
+import { useState } from 'react';
+
+import { useRouter } from 'next/navigation';
+
+import { Monitor, Moon, Sun } from 'lucide-react';
+import { useTheme } from 'next-themes';
+import { useHotkeys } from 'react-hotkeys-hook';
+
+import {
+  CommandDialog,
+  CommandEmpty,
+  CommandGroup,
+  CommandInput,
+  CommandItem,
+  CommandList,
+  CommandShortcut,
+} from '@documenso/ui/primitives/command';
+
+export function CommandMenu() {
+  const { setTheme } = useTheme();
+  const { push } = useRouter();
+  const [open, setOpen] = useState(false);
+  const [search, setSearch] = useState('');
+  const [pages, setPages] = useState<string[]>([]);
+  const page = pages[pages.length - 1];
+
+  useHotkeys('ctrl+k', (e) => {
+    e.preventDefault();
+    setOpen((open) => !open);
+  });
+  useHotkeys('n+s', () => push('/settings'));
+  useHotkeys('n+d', () => push('/documents?status=ALL'));
+
+  return (
+    <CommandDialog
+      commandProps={{
```

Comment:
We could probably do something with typescript here to accept the subset of `commandbar` props we're interested in but I'm happy to stick with this for now.